reporter/src/main/kotlin/model/Stats.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import java.util.SortedSet
+
+/**
+ * A class containing statistics for an [OrtResult].
+ */
+data class Stats(
+    /**
+     * The number of [OrtIssue]s by severity.
```

Comment:
In the next commit it becomes clear that this contains only unresolved issues. This should be documented here, and maybe the field renamed to `unresolvedIssues`. Wouldn't it be interesting to also have the stats for resolved issues?
Same for the violations below.